runtime/near-vm-runner/src/imports.rs

```diff
@@ -284,6 +284,13 @@ imports! {
     // #  Sandbox  #
     // #############
     ##["sandbox"] sandbox_debug_log<[len: u64, ptr: u64] -> []>,
+
+    // Sleep for the given number of nanoseconds. This is the ultimate
+    // undercharing function as it doesn't consume much gas but takes a lot of
+    // time to execute. It must always be gated behind a feature flag and it
+    // must never be released to production.
+    // TODO!!! Hide this behind a feature flag!
```

Comment:
nit. I guess adding this TODO to a single place would suffice, maybe here.

Reply:
Those need to be addressed before merging, I'll do that shortly.